src/_gettsim_tests/test_data/parent_child_links/2007/eltern_getrennt_1_kind.yaml

```diff
@@ -0,0 +1,113 @@
+---
+info:
+  note: ''
+  source: ''
+inputs:
+  provided:
+    p_id:
+      - 3
+      - 4
+      - 5
+    geburtstag:
+      - 1
+      - 1
+      - 1
+    geburtsmonat:
+      - 1
+      - 1
+      - 1
+    geburtsjahr:
+      - 1970
+      - 1970
+      - 2004
+    # TODO (@hmgaudecker): Remove again.
+    # https://github.com/iza-institute-of-labor-economics/gettsim/issues/211
+    alter:
+      - 37
+      - 37
+      - 3
+    hh_id:
+      - 1
+      - 1
+      - 1
+    p_id_einstandspartner:
+      - -1
+      - -1
+      - -1
+    p_id_ehepartner:
+      - -1
+      - -1
+      - -1
+    gemeinsam_veranlagt:
+      - false
+      - false
+      - false
+    p_id_elternteil_1:
+      - -1
+      - -1
+      - 3
+    p_id_elternteil_2:
+      - -1
+      - -1
+      - 4
+    alleinerz:
+      - true
+      - false
+      - false
+    eigener_bedarf_gedeckt:
+      - false
+      - false
+      - false
+    p_id_kindergeld_empfänger:
+      - -1
+      - -1
+      - 3
+    p_id_kinderfreibetrag_empfänger_1:
+      - -1
+      - -1
+      - 3
+    p_id_kinderfreibetrag_empfänger_2:
+      - -1
+      - -1
+      - 3
+    p_id_erziehungsgeld_empfänger:
+      - -1
+      - -1
+      - 3
+    kindergeld_anspruch:
```

Comment:
Shouldn't this be endogenous?

Reply:
Yes, that can be based on `p_id_elternteil_x`.